### ISSUE TITLE
ik_marker_errors.sto and ik_model_marker_locations.sto are hardcoded filenames in IKTool

### ISSUE THREAD

**Issue by marnunez** [opensim-org/opensim-core#2564]
This is quite troublesome for batch processing with python/MATLAB, where one has mutiple .trc files for a single model file and many IKtools running in paralell,  where the last one to finish overwrites the ik_marker_errors.sto and ik_model_marker_locations.sto files.

**Comment by jimmyDunne** [comment]
I don't know if that is a viable solution. If you are at the point where you need to run IK in parallel, IMHO, you should be using the IK solver directly. 

There is a rough example of using the solver [here](https://github.com/jimmyDunne/opensim-matlab/blob/71b425d024d9015054f46c6b8d1f2a9beeaa559e/solvers/solverIK.m)

**Comment by marnunez** [comment]
Oh no, I wasn't describing a solution, I was describing the situation. "ik_marker_errors" and "ik_model_marker_locations" are hardcoded filenames in the IKTool. Overwriting the errors file is not an intended outcome.
Perhaps a <report_errors_filename> and a <report_marker_locations_filename> tag could do the trick.

**Comment by jimmyDunne** [comment]
Yep, I know they are hardcoded. My point is that the IKTool shouldn't be used by people who are doing computations in parallel. All the tools were specifically designed to be used by the GUI and command line and have been retrofitted to _kind-of-work_ through scripting. You shouldn't be relying on them as you are. Use the IKSolver so you can get fine grain control on what you want to do and the outputs from the model.

**Comment by marnunez** [comment]
Thanks @jimmyDunne , I'll look into it, it's time for a rewrite.

**Comment by jimmyDunne** [comment]
Thanks for posting an issue! We will keep it open so (hopefully) we can update the tool someday ;)
